tests/fixtures/custom_task_interface_multiclass/custom.py

```diff
@@ -53,4 +55,6 @@ def predict_proba(self, X, **kwargs):
         pd.DataFrame
             Returns a dataframe with transformed data.
         """
+
+        # Note how multiclass estimators require one column per class
```

Comment:
add comment about column names

Reply:
@jmbannon sorry, which comment?

Reply:
i mean on this line mention something about how we have to pass column names derived from the estimator